CGPs/cgp-0097.md

```diff
@@ -1,12 +1,12 @@
 ---
 cgp: 97
-title: MU03: Enable more trading pairs and continue phasing out V1
```

Comment:
strings in yaml like this cannot contain `:` chars or they'll be treated as objects